.env.template

```diff
@@ -1,6 +1,13 @@
-THOTH_CEPH_BUCKET=
-THOTH_CEPH_BUCKET_PREFIX=
+# source ceph
+THOTH_DEPLOYMENT_NAME="ocp-test"
+
+THOTH_S3_ENDPOINT_URL="https://s3.upshift.redhat.com/
+THOTH_CEPH_BUCKET="thoth"
+THOTH_CEPH_BUCKET_PREFIX="data"
 THOTH_CEPH_KEY_ID=
 THOTH_CEPH_SECRET_KEY=
```

Comment:
Don't we need credentials with this ? from python or aws-cli I get the same error -> Unable to locate credentials.

Reply:
yes we do, you can get the credential from the cluster.
https://console-openshift-console.apps.smaug.na.operate-first.cloud/k8s/ns/thoth-frontend-prod/secrets/ceph

I would like to include these detail in thoth-application docs,
so I have included the link to this in this document.

Reply:
Right, this ought to be in a more general place than here :+1: